pkg/watches/watches.go

```diff
@@ -33,13 +33,13 @@ import (
 
 type Watch struct {
 	schema.GroupVersionKind `json:",inline"`
-	ChartPath               string `json:"chart"`
+	// Though the name is ChartDir, this accepts chart directory or a chart path.
+	ChartDir string `json:"chart"`
 
-	WatchDependentResources *bool             `json:"watchDependentResources,omitempty"`
-	OverrideValues          map[string]string `json:"overrideValues,omitempty"`
-	ReconcilePeriod         *metav1.Duration  `json:"reconcilePeriod,omitempty"`
-	MaxConcurrentReconciles *int              `json:"maxConcurrentReconciles,omitempty"`
-	Chart                   *chart.Chart      `json:"-"`
+	WatchDependentResources *bool                `json:"watchDependentResources,omitempty"`
+	OverrideValues          map[string]string    `json:"overrideValues,omitempty"`
+	Chart                   *chart.Chart         `json:"-"`
+	Selector                metav1.LabelSelector `json:"selector"`
```

Comment:
Really minor nit: move `Selector` above `Chart` to keep all of the fields that are present in the JSON format together.